packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -154,14 +102,34 @@ information, see the States Language spec.
 
 ### Task
 
-A `Task` represents some work that needs to be done. It takes a `resource`
-property that is either a Lambda `Function` or a Step Functions `Activity`
-(A Lambda Function runs your task's code on AWS Lambda, whereas an `Activity`
-is used to run your task's code on an arbitrary compute fleet you manage).
+A `Task` represents some work that needs to be done. The exact work to be
+done is determine by a class that implements `IStepFunctionsTask`, a collection
+of which can be found in the `@aws-cdk/aws-stepfunctions-tasks` package. A
+couple of the tasks available are:
+
+* `tasks.InvokeFunction` -- call a Lambda Function
+* `tasks.InvokeActivity` -- start an Activity (Activities represent a work
+  queue that you poll on a compute fleet you manage yourself)
+* `tasks.PublishToTopic` -- publish a message to an SNS topic
+* `tasks.SendToQueue` -- send a message to an SQS queue
+* `tasks.EcsRunFargateTask`/`ecs.EcsRunEc2Task` -- run a container task,
```

Comment:
I think `EcsRun*Task` should be renamed to `RunEcsFargateTask` and `RunEcsEc2Task`. Seems like the verb comes first in other tasks (which I like :-)).

Reply:
Had that at first but I hated the `EcsEc2` part of it. But you're right, it IS more consistent.